README.md

```diff
@@ -1,44 +1,44 @@
 
 **Nextjs**
 
-```cli
+"`cli
 $ yarn install
 $ yarn dev
+
 ```
 
 **Gatsby**
 
-```cli
+"`cli
 $ npm install
 $ npm run develop
 ```
 
-Afther yarn is ready with setting up your dev environment you should be able to navigate to [http://localhost:3000/](http://localhost:3000/) on Next or to [http://localhost:8000/](http://localhost:8000/) for gatsby and see the same application as on
-[https://conductionnl.github.io/nl-design-skeleton-gatsby/](https://conductionnl.github.io/nl-design-skeleton-gatsby/).  Feel free to open up your project code and make some changes you should see the changes in your development environment isntantainusly.
+After Yarn has successfully set up your dev environment, navigate to [http://localhost:3000/](http://localhost:3000/) on Next.js or to [http://localhost:8000/](http://localhost:8000/) for Gatsby and see the same application as on
+[https://conductionnl.github.io/nl-design-skeleton-gatsby/](https://conductionnl.github.io/nl-design-skeleton-gatsby/). Feel free to open up your project code and make some changes. You should see the changes in your development environment instantaneously due to hot-reloading.
 
 ## Publishing your prototype to the internet (gatsby only)
 
-The gatsby version of the skeleton application has build in support for github pages, meaning that you can build your application into a static website and publish it as a github page. The skeleton repository comes with a build in github action for publishing itsself as a github page, meaning that you can have your prototype automatically published to the internet on a code push.
+The Gatsby version of the skeleton application has build-in support for GitHub pages, meaning you can build your application into a static website and publish it as a Github page. The skeleton repository comes with a build-in GitHub action for publishing itself as a GitHub page. You can have your prototype automatically published to the internet on a code push this way.
 
-For this to work you will need to do active github-pages on your repository, go to your repository settings and click on pages select gh-pages as a source and press on save (if you do not see a gh-pages branch yet you can create one by pushing to master).
+You need to activate GitHub pages on your repository for this to work. Go to your repository's settings, go to pages, select `gh-pages` as a source, and save (if you do not see a gh-pages branch, you can create one by pushing to main).
 
-After clicking on save you can wait for github to publish your project and provide  you with al link you can share for your demo. Keep in mind that from this point on all pushes to master will result in updates to your online demo environment.
+After saving, wait for GitHub to publish your project and provide you with a link you can share for your demo. Keep in mind that all pushes to main will result in updates to your online demo environment from this point on.
```

Comment:
Misschien dat ik het niet goed begrijp, maar: volgens mij geven we net aan dat we onze GitHub page op de branch `gh-pages` willen plaatsen. Nu zeggen we dat wanneer we naar `main` pushen dat de live omgeving geüpdatet wordt. Zou het niet zo moeten zijn dat wanneer we naar `gh-pages` pushen dat het update?

Wederom: misschien zit ik er naast, maar denk wel goed om nog even te dubbel checken!

Reply:
daarom dus Gatsby. Die doet dat automatisch. Anders moet je idd branch aangeven.